src/app/controllers/message.js

```diff
@@ -322,13 +383,32 @@ Message.prototype.deleteBox = function(boxName){
     })
 }
 
-Message.prototype.retrieveMessage = function(id, boxName){
+Message.prototype.retrieveMessage = function(id, boxName, isUnread){
```

Comment:
Menggunakan boolean di argumen fungsi sangat tidak disarankan(*). Ini apa arti `isUnread` sebenarnya?

*. http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html